activerecord/test/cases/primary_keys_test.rb

```diff
@@ -224,7 +224,9 @@ class Barcode < ActiveRecord::Base
   end
```

Comment:
Using `if_exists: true` allows us to remove `table_exists?` here, I will change.